src/Collection/Api/Mastodon/Mentions.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Friendica\Collection\Api\Mastodon;
+
+use Friendica\BaseCollection;
+use Friendica\Object\Api\Mastodon\Mention;
+
+class Mentions extends BaseCollection
+{
+	/**
+	 * @return Mention
+	 */
+	public function current()
```

Comment:
Can you add the return value hint or would this break inheritance rules?
```suggestion
	public function current(): Mention
```

Reply:
no, I didn't know that PHP 7.0 supports the return value hints --> I would have had used it before ...